docs/doc/10-deploy/00-understanding-deployment-modes.md

```diff
@@ -12,16 +12,20 @@ A Databend deployment includes two types of node, Meta and Query. The Meta node
 When deploying Databend, you specify a deployment mode, standalone, or cluster. A standalone Databend allows for one Meta node and one Query node, and a Databend cluster can include multiple Meta(databend-meta) to archive high availability and Query(databend-query) nodes to enhance the computing capability.
 
 ### Supported Object Storage Solutions
-Databend works with self-hosted or cloud object storage solutions. You will need to prepare your own object storage before deploying Databend. The following is a list of compatible object storage solutions:
+Databend supports for self-hosted and cloud object storage solutions. Prepare your own object storage before deploying Databend. The following is a list of supported object storage solutions:
+
 - AWS S3 Compatible Services:
   - Amazon S3
   - MinIO
+  - Ceph
   - Wasabi
   - SeaweedFS
   - Tencent COS
   - Alibaba OSS
```

Comment:
We also have native oss support now.

Reply:
What are the differences between native OSS and S3?

Reply:
Native OSS uses OSS's native API, supports integrate with [RAM](https://www.aliyun.com/product/ram) and so on.

Access oss with s3 uses OSS's S3 compatible API.